### PR TITLE
Added date and time to the list of recently edited resources in the personal profile

### PR DESCRIPTION
**Pull request by igorsuhinin**
### What does it do?
Adds date and time to the list of recently edited resources in the personal profile.

### Why is it needed?
Currently the list of recently edited resources doesn't show the date & time when the resource was edited.  There is also a small bug: the list of YOUR recently edited resources in the personal profile shows the most recently edited resources without filtering by the current user ID.

### Related issue(s)/PR(s)
Issue #11264 
PR #14511

### CODE REVIEW

File: core/model/modx/processors/security/user/getrecentlyeditedresources.class.php [hunk 1]
```diff
@@ -58,8 +58,13 @@ public function initialize()
      */
     public function prepareQueryBeforeCount(xPDOQuery $c)
     {
+        $user = $this->getProperty('user');
         $q = $this->modx->newQuery($this->classKey, ['classKey:IN' => $this->classKeys]);
         $q->select('MAX(id), item');
+        if (!empty($user)) {
+            $q->where(['user' => $user]);
+            $c->where(['user' => $user]);
```
Comment (alroniks) [inline core/model/modx/processors/security/user/getrecentlyeditedresources.class.php:66]: Is it required here?

Comment (alroniks) [reply core/model/modx/processors/security/user/getrecentlyeditedresources.class.php:66]: Seems like accidentally duplicated lines. Need to leave the first one.

Comment (igorsuhinin) [reply core/model/modx/processors/security/user/getrecentlyeditedresources.class.php:66]: > `Seems like accidentally duplicated lines. Need to leave the first one`

Not really, both are required. Please note, there are two different variables **_$q_** and **_$c_**.

I just tried to make as less changes as I could. So I just used the variables which were added before.

Comment (alroniks) [reply core/model/modx/processors/security/user/getrecentlyeditedresources.class.php:66]: Ok, I see now. 👍

**Review by Mark-H — APPROVED** [review]

**Comment by Mark-H** [comment]
FYI This PR also includes changes already proposed in #14511.

**Pull request merged**